common/src/device/provisioning/transport/DeviceRegistrationResult.cs

```diff
@@ -14,128 +14,64 @@ namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Models
-        /// </summary>
         [JsonProperty(PropertyName = "x509")]
         public X509RegistrationResult X509 { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "symmetricKey")]
         public SymmetricKeyRegistrationResult SymmetricKey { get; set; }
 
-
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "registrationId")]
         public string RegistrationId { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "createdDateTimeUtc")]
         public DateTime? CreatedDateTimeUtc { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "assignedHub")]
         public string AssignedHub { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "deviceId")]
         public string DeviceId { get; set; }
 
         /// <summary>
-        /// Gets or sets possible values include: 'unassigned', 'assigning',
-        /// 'assigned', 'failed', 'disabled'
+        /// The status of the operation.
+        /// Possible values include: 'unassigned', 'assigning', 'assigned', 'failed', 'disabled'
         /// </summary>
         [JsonProperty(PropertyName = "status")]
         public string Status { get; set; }
 
         /// <summary>
+        /// The substatus of the operation.
         /// Possible values include: 'initialAssignment', 'deviceDataMigrated', 'deviceDataReset'
         /// </summary>
         [JsonProperty(PropertyName = "substatus")]
         public string Substatus { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "generationId")]
         public string GenerationId { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "lastUpdatedDateTimeUtc")]
         public DateTime? LastUpdatedDateTimeUtc { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "errorCode")]
         public int? ErrorCode { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "errorMessage")]
         public string ErrorMessage { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "etag")]
         public string Etag { get; set; }
 
-        /// <summary>
-        /// </summary>
         [JsonProperty(PropertyName = "payload")]
         public JRaw Payload { get; set; }
+
+        /// <summary>
+        /// The PEM encoded operational client certificate that was returned by the CA. This operational certificate was used by DPS to register the enrollment with IoT Hub.
+        /// The IoT device can use this operational certificate to authenticate with IoT Hub.
+        /// </summary>
+        [JsonProperty(PropertyName = "issuedClientCertificate")]
+        public string IssuedClientCertificate { get; set; }
```

Comment:
Is there any reason why this needs to have a public setter? It sounds like this is more of a read-only field

Reply:
You're right, this is a read-only field and should not require a public setter. However, newtonsoft json requires the properties to be public for it to be able to deserialize them.